workflow/scripts/neighborhoods.py

```diff
@@ -410,17 +410,18 @@ def get_class_pyranges(
 
 
 def run_count_reads(target, output, bed_file, genome_sizes, use_fast_count):
-    if target.endswith(".bam"):
+    filename = os.path.basename(target)
+    if filename.endswith(".bam"):
         count_bam(
             target,
             bed_file,
             output,
             genome_sizes=genome_sizes,
             use_fast_count=use_fast_count,
         )
-    elif target.endswith(".tagAlign.gz") or target.endswith(".tagAlign.bgz"):
+    elif "tagAlign" in filename:
```

Comment:
with igvf datasets, we can't assume it ends with tagalign

Reply:
I'd look into the files with bgz suffix, unless all the files have bgz as the suffix.

Reply:
what do you mean by 'look into'? I think all files are typically just '.gz', but it can be '...tagAlign.tsv.gz', so we were getting errors before as the code didn't recognize that as a tagAlign file